partcad/src/partcad/part_factory_extrude.py

```
@@ -61,19 +56,43 @@ def __init__(self, ctx, source_project, target_project, config):
+                    raise Exception(errors)
 
-            self.ctx.stats_parts_instantiated += 1
+                result = shape_envelope.deserialize(response_serialized)
+                if not result["success"]:
+                    part.error("%s: %s" % (part.name, result["exception"]))
+                    return None
 
-            return shape
+                self.ctx.stats_parts_instantiated += 1
+                return result["shape"]
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)